### ISSUE TITLE
Reduce size by 96 bytes

### ISSUE THREAD

**Issue by bradanlane** [technoblogy/tiny-tft-graphics-library#2]
The `CharMap` is 96*6 bytes but the 6th byte of each character is always 0x00.

You can delete these 6th bytes and then change `PlotChar()` to loop from 0 to 5 for `xx`.

**Comment by technoblogy** [comment]
Yes, that's a good suggestion if you need those extra bytes. I left the 0x00 there in case you want to define full 6x8 pixel characters; eg graphics characters.
